components/EventForm/index.js

```diff
@@ -3,63 +3,86 @@ import { useState } from "react";
 import { useRouter } from "next/router";
 import Modal from "../ModalWindow";
 
+const Form = styled.form`
+  display: flex;
+  flex-direction: column;
+  height: 80vh;
+  padding: 20px;
+  font-family: monospace;
+`;
+
 const Paragraph = styled.p`
   font-size: small;
   color: gray;
   margin: 10px;
 `;
 
+const Label = styled.label`
+  font-size: 1rem;
+  margin-bottom: 5px;
+  align-self: flex-start;
+  @media (min-width: 501px) {
+    font-size: 1.2rem;
+  }
+`;
+
 const Input = styled.input`
   font-family: monospace;
   border-radius: 8px;
   outline: 3px;
-  height: 35px;
-  width: 300px;
+  height: 40px;
   border: 0px;
+  width: 80vw;
   background-color: #d1c4e9;
   padding: 15px;
-  margin-bottom: 20px;
+  margin: 0px auto 20px;
   box-shadow: rgba(0, 0, 0, 0.15) 1.95px 1.95px 2.6px;
 
   &:focus {
     background-color: whitesmoke;
   }
+  @media (min-width: 501px) {
+    height: 45px;
+    max-width: 500px;
+  }
 `;
 
 const Textarea = styled.textarea`
+  width: 100%;
   font-family: monospace;
   border-radius: 8px;
   background-color: #d1c4e9;
   border: 0;
   padding: 5px;
+  height: 40px;
+  width: 80vw;
   box-shadow: rgba(0, 0, 0, 0.15) 1.95px 1.95px 2.6px;
 
   &:focus {
     background-color: #f5f5f5;
   }
+  @media (min-width: 501px) {
+    height: 45px;
+    max-width: 500px;
+  }
 `;
 const Submit = styled.button`
+  padding: 10px;
+  margin-top: 10px;
   border-radius: 8px;
   border: 0;
   background-color: #bdbdbd;
   font-family: monospace;
   height: 40px;
+  width: 160px;
   color: black;
```

Comment:
See above, color codes are preferred.

```suggestion
  color: #000000;
```